package/MDAnalysis/analysis/hbonds/hbond_autocorrel.py

```diff
@@ -427,11 +430,10 @@ def _single_run(self, start, stop):
         box = self.u.dimensions if self.pbc else None
 
         # 2d array of all distances
-        pair, d = capped_distance(self.h.positions, self.a.positions, max_cutoff=self.d_crit, box=box)
-        if self.exclusions:
-            # set to above dist crit to exclude
-            exclude = np.column_stack((self.exclusions[0], self.exclusions[1]))
-            pair = np.delete(pair, np.where(pair==exclude), 0)
+        pair = capped_distance(self.h.positions, self.a.positions, max_cutoff=self.d_crit, box=box,
+                               return_distances=False)
+        if not self.exclusions is None:
```

Comment:
Equivalent and more common in Python:
```suggestion
        if self.exclusions is not None:
```